third_party/ent/level_update.go

```diff
@@ -46,6 +47,12 @@ func (lu *LevelUpdate) SetDescription(s string) *LevelUpdate {
 	return lu
 }
 
+// SetDefinition sets the definition field.
```

Comment:
Consulta sobre ent, estos files te los genera solos? O con cada field nuevo tenes que tocarlos?

Reply:
Lo genera todo solo. Hay un generate command (lo puse en `generate.go`). Vos interactuas solamente con los files de `ent/schema/`, una vez que modificaste lo que necesitabas, corres el generate y te actualiza todo esto automaticamente.

Todos los files que empiecen con `// Code generated by entc, DO NOT EDIT.` son... generados por ent